proposals/sar_backscatter.json

```diff
@@ -0,0 +1,157 @@
+        {
+            "name": "elevation_model",
+            "description": "The digital elevation model to use. Set to `null` (the default) to allow the back-end to choose, which will improve portability, but reduce reproducibility.",
+            "optional": true,
+            "default": null,
+            "schema": [
+                {
+                    "type": "string",
+                    "subtype": "collection-id"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        {
+            "name": "mask",
+            "description": "If set to `true`, a data mask is added to the bands. It indicates which values are valid (1), invalid (0) or contain no-data (null).",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "contributing_area",
+            "description": "If set to `true`, a DEM-based local contributing area band is added. The values are given in square meters.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "local_incidence_angle",
+            "description": "If set to `true`, a DEM-based local incidence angle band is added. The values are given in degrees.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "ellipsoid_incidence_angle",
+            "description": "If set to `true`, a ellipsoidal incidence angle band is added. The values are given in degrees.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "gamma_sigma",
+            "description": "If set to `true`, a band is added that provides the ratio of the integrated area in the Gamma projection over the integrated area in the Sigma projection (ground).",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "date_offset",
+            "description": "If set to `true`, a band is added that provides the acquisition date for each pixel. Values are day offset to the first observation date (in UTC) as specified in the metadata.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "scaling",
+            "description": "By default, all values are given in logarithmic decibel (dB) scale. If set to `linear`, the values are given in linear scale. Values are converted from linear to dB using *dB = 10*log₁₀(l)* and vice-versa.",
```

Comment:
`*dB = 10*log₁₀(l)*`

I think the three asterisks here will generate wrong markup. The subscript 10 also looks risky in terms of font/tool support, I think `log10` is just fine. The `l` argument of log10 is also to avoid: it's too close to upper case `i` or `1` in some fonts, e.g. just use `linear` (as you use `dB` at the left hand side).

Reply:
The formula has been removed due to another review comment.